src/Me/Profile/Profile.js

```diff
@@ -12,6 +12,8 @@ import {
   clearCurrentUser,
 } from '../../api/index';
 import messages from '../../messages';
+import ISO6391 from 'iso-639-1';
+import { getName } from 'country-list';
```

Comment:
We have the counties name from `svg-country-flags`. Like this

https://github.com/Coding-Coach/find-a-mentor/blob/96d45d96c3ef7d0ef8ba17e9c4f91892a5f21aa5/src/listsGenerator.js#L1

Reply:
I have changed it @moshfeu